guide/daily-spending-wallet/backup-and-recovery/recovery.md

```diff
@@ -33,47 +33,395 @@ Illustration sources:
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-recovery-phrase-complete@2x.png"
+   layout = "shadow"
+   caption = "The user has finished entering their recovery phrase and taps \"Next\"."
+   alt-text = "Screen showing a 12 word recovery phrase in green text"
+   width = 250
+   height = 541
+%}
 
-{% include tip/close.html %}
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-channel-selection.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-channel-selection@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-channel-selection@2x.png"
+   layout = "shadow"
+   caption = "The app fails to automatically find a channel state backup in the user's cloud account. User chooses to try another cloud account."
+   alt-text = "Screen describing how a channel backup couldn't be found, with a list of options"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-cloud-select.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-cloud-select@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-cloud-select@2x.png"
+   layout = "shadow"
+   caption = "User chooses their cloud provider."
+   alt-text = "Screen showing a list of cloud provider choices."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-cloud-ui.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-cloud-ui@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-cloud-ui@2x.png"
+   layout = "shadow"
+   caption = "User authenticats with the cloud provider, finds the backup file, etc."
+   alt-text = "Screen showing a placeholder image for a cloud provider UI"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-checking-file.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-checking-file@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-checking-file@2x.png"
+   layout = "shadow"
+   caption = "The app checks the backup file the user has selected."
+   alt-text = "Screen showing a backup file being checked"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-checking-complete.png"
```

Comment:
This image looks identical to the previous one...?

Reply:
The other says "checking", this one says "looks good". I can strip out the other one if this is too much detail.

Reply:
Ah, I see it now. Maybe the second one could have green text and a checkmark or something like that?

Reply:
Done